block/block.go

```diff
@@ -140,13 +141,17 @@ func (m *Manager) attemptApplyCachedBlocks() error {
 		m.logger.Debug("applied cached block", "height", expectedHeight)
 	}
 
+	return nil
```

Comment:
I removed the pruning from each gossiped block, as it's not efficient, it goes over **all** the cached blocks.